cdc/cdc_options.hh

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2019 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <map>
+#include <seastar/core/sstring.hh>
+
+namespace cdc {
+
+using namespace seastar;
```

Comment:
Unfortunately all our code is written assuming `using namespace sestar`. This is a relic of our history, from the past where `seastar` wasn't yet a separate framework, just a part of the Scylla codebase.

Reply:
Indeed. Many of our header files already have "using namespace seastar;" so if you include any one of those, you don't need to do this yourself, but sometimes in new header files which don't include a lot of header files - you do. Even then we recommend to get the "using namespace seastar" by doing "#include "seastarx.hh", not by directly calling "using namespace". All of this is documented in docs/coding-style.md (in fact, it's more or less the only thing documented there, since the rest of the coding style is inherited from Seasatar).